exporters/variables-scss/src/content/generators/fileGenerator.ts

```diff
@@ -0,0 +1,56 @@
+import { TokenGroup, Token, TokenType } from '@supernovaio/sdk-exporters';
+import { generateFileContent } from './contentGenerator';
+
+const filesData = [
+  {
+    fileName: '_spacing.scss',
+    tokenTypes: TokenType.dimension,
```

Comment:
You are using the plural here for a single value. That suggests that there should be more values, an array.

From your code, I see you are transforming these values into an array. So making it an array here will simplify your code.